web/app/view/Report.js

```diff
@@ -67,17 +67,50 @@ Ext.define('Traccar.view.Report', {
         }]
     },
 
-    listeners: {
-        selectionchange: 'onSelectionChange'
+    layout: {
+        type: 'fit'
     },
 
-    forceFit: true,
-
-    columns: {
-        defaults: {
-            minWidth: Traccar.Style.columnWidthNormal
+    items: [{
+        xtype: 'grid',
+        itemId: 'grid',
+        listeners: {
+            selectionchange: 'onSelectionChange'
+        },
+        hidden: true,
+        forceFit: true,
+        flex: 1,
+        columns: {
+            defaults: {
+                minWidth: Traccar.Style.columnWidthNormal
+            },
+            items: [
+            ]
+        },
+        style: 'borderTop: 1px solid lightgray'
```

Comment:
Unfortunately not. Line is there until select some type. 

![image](https://cloud.githubusercontent.com/assets/5688080/21490819/11263362-cc1a-11e6-915a-f77dc8504129.png)